unit-tests/live/d400/test-emitter-frequency.py

```diff
@@ -0,0 +1,63 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#test:device:jetson D457
+#test:device:!jetson D455
```

Comment:
What should happen with jetson, and other devices than D457 and D455?

Reply:
The test will not run, and the option will not be registered.

Reply:
ok